### PR TITLE
22223 Categorize classes in System-Changes

### PR DESCRIPTION
**Pull request by astares**
https://pharo.fogbugz.com/f/cases/22223/Categorize-classes-in-System-Changes

### CODE REVIEW

**Comment by jecisc** [comment]
This cause failing tests:

unix-32 / Tests-unix-32 / testExternalBasicToolsDependencies – Unix32.System.DependenciesTests.SystemDependenciesTest<1s
unix-32 / Tests-unix-32 / testExternalIDEDependencies – Unix32.System.DependenciesTests.SystemDependenciesTest<1s
unix-32 / Tests-unix-32 / testExternalLocalMonticelloDependencies – Unix32.System.DependenciesTests.SystemDependenciesTest<1s
unix-32 / Tests-unix-32 / testExternalMonticelloDependencies – Unix32.System.DependenciesTests.SystemDependenciesTest<1s
unix-32 / Tests-unix-32 / testExternalMorphicDependencies – Unix32.System.DependenciesTests.SystemDependenciesTest<1s
unix-32 / Tests-unix-32 / testExternalUIDependencies – Unix32.System.DependenciesTests.SystemDependenciesTest<1s
osx-32 / Tests-osx-32 / testExternalBasicToolsDependencies – MacOSX32.System.DependenciesTests.SystemDependenciesTest<1s
osx-32 / Tests-osx-32 / testExternalIDEDependencies – MacOSX32.System.DependenciesTests.SystemDependenciesTest<1s
osx-32 / Tests-osx-32 / testExternalLocalMonticelloDependencies – MacOSX32.System.DependenciesTests.SystemDependenciesTest<1s
osx-32 / Tests-osx-32 / testExternalMonticelloDependencies – MacOSX32.System.DependenciesTests.SystemDependenciesTest<1s
osx-32 / Tests-osx-32 / testExternalMorphicDependencies – MacOSX32.System.DependenciesTests.SystemDependenciesTest<1s
osx-32 / Tests-osx-32 / testExternalUIDependencies – MacOSX32.System.DependenciesTests.SystemDependenciesTest<1s
windows-32 / Tests-windows-32 / testExternalBasicToolsDependencies – Windows32.System.DependenciesTests.SystemDependenciesTest<1s
windows-32 / Tests-windows-32 / testExternalIDEDependencies – Windows32.System.DependenciesTests.SystemDependenciesTest<1s
windows-32 / Tests-windows-32 / testExternalLocalMonticelloDependencies – Windows32.System.DependenciesTests.SystemDependenciesTest<1s
windows-32 / Tests-windows-32 / testExternalMonticelloDependencies – Windows32.System.DependenciesTests.SystemDependenciesTest<1s
windows-32 / Tests-windows-32 / testExternalMorphicDependencies – Windows32.System.DependenciesTests.SystemDependenciesTest<1s
windows-32 / Tests-windows-32 / testExternalUIDependencies – Windows32.System.DependenciesTests.SystemDependenciesTest

**Comment by astares** [comment]
Failing tests were green in image ... lets see what CI says now.

**Comment by jecisc** [comment]
Did you run them before the changes? Because I think those tests caches the dependencies computation since it takes a long time. So maybe you run the tests with the cached dependencies.

File: src/System-Changes/ChangeSet.class.st [hunk 1]
```diff
@@ -186,6 +186,21 @@ ChangeSet class >> doesAnyChangeSetHaveClass: aClass andSelector: aSelector [
 	^ (self countOfChangeSetsWithClass: aClass andSelector: aSelector) > 0
 ]
 
+{ #category : #'as yet unclassified' }
```
Comment (jecisc) [inline src/System-Changes/ChangeSet.class.st:189]: Method is unclassified

Comment (astares) [reply src/System-Changes/ChangeSet.class.st:189]: I categeorized only the classes - not the methods here

**Comment by astares** [comment]
No I did not run them before and used a fresh image. Lets see what CI tells us...

**Comment by astares** [comment]
Requires integration of https://pharo.fogbugz.com/f/cases/22445/Cleanup-System-CommandLine-Manifest-package-and-manifest first to get CI green again

**Comment by astares** [comment]
Continued in https://github.com/pharo-project/pharo/pull/1930

**Pull request closed without merging**